qiita_db/study.py

```diff
@@ -206,6 +207,26 @@ def get_by_status(cls, status):
 
         return studies
 
+    @staticmethod
+    def get_by_portal(portal):
```

Comment:
This is here because it will allow much fewer changes to the code base. For example, we can just add a single set intersection check instead of modifying every single SQL query interacting with the studies.